kythe/docs/schema/schema.txt

```diff
@@ -1201,6 +1201,34 @@ public class E {
 }
 --------------------------------------------------------------------------------
 
+[[refid]]
+ref/id
+~~~~~~~~~~~
+Brief description::
+  A *ref/id* B if A does not otherwise *ref* or *define* B but the bytes of A's
+  span are determined by the identifier of B.
+Points from::
+  anchors
+Points toward::
+  semantic nodes
+Ordinals are used::
+  never
+
+[kythe,C++,"Constructors are named after their classes."]
```

Comment:
Do you have a second example?  (Why isn't this just 'ref/ctorOf'?)

Reply:
what about "Constructors and destructors are named after their classes." ?

Reply:
This is a more general relationship than just constructor-/destructor-of.

Reply:
Right, I get that it's intended to be more general, but with only a single example I am not sure what the generalization is.  When would I use id instead of some other relationship?  Is it meant as a catch-all for cases where you don't have constructor-of?  Etc.

Reply:
At the moment, no other use cases have arisen but we try not to paint ourselves into a corner.  Edges and nodes are generally given names that describe the semantics specifically enough to differentiate them, but without being overly language specific.

Reply:
Just making sure I follow, I should be using this for any connections that aren't covered by the existing schema then?